### DISCUSSION TITLE
Product version of latest release

### DISCUSSION THREAD

**Discussion by Emil-Granbom** [mganss/HtmlSanitizer#256] [General]
Why is the product version of the dll files in HtmlSanitizer.5.0.372 1.0.0? It triggers vulnerability warnings in our static analysis tool (Coverity from Synopsis).

![image](https://user-images.githubusercontent.com/69844503/104694515-97532f00-570b-11eb-9fb0-0b20ecafa57f.png)

**Comment by mganss** [comment]
This was because `InformationalVersion` wasn't set. [.NET open source library guidance](https://docs.microsoft.com/en-us/dotnet/standard/library-guidance/versioning#assembly-informational-version) says:

> If you're using Source Link, this version will be set on build with the NuGet package version plus a source control version.

plus 

> ❌ AVOID setting the assembly informational version yourself.

Unfortunately, this doesn't work.

I've added `InformationalVersion` in 5.0.376.
